test/test.py

```diff
@@ -152,6 +152,53 @@ def test_dl_transpose4(self):
         self.assertEqual(self._dl('bca', 'ab'), 2.5)
 
 
+class TestClev_noinit(unittest.TestCase):
```

Comment:
Classes in Python usually shouldn't have underscores in the name. I'm also not sure that `noinit` communicates what's going on in this test as well as something like `TestClevUsingDefaultValues`.